### PR TITLE
Add map to mission index

### PR DESCRIPTION
**Pull request by Nicode80**
Add map to missions index. The map focus on current user position.

!! I have also redirected to this page so home not available any more !!
Capture d’écran 2020-11-24 à 22 46 34

### CODE REVIEW

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -2,5 +2,6 @@ class PagesController < ApplicationController
   skip_before_action :authenticate_user!, only: [ :home ]
 
   def home
+    redirect_to missions_path
```
Comment (mdaudiffret) [inline app/controllers/pages_controller.rb]: Why do you redirect to missions index? shouldn't it be campaigns index?

File: app/javascript/plugins/init_mapbox.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ const initMapbox = async () => {
   mapboxgl.accessToken = mapElement.dataset.mapboxApiKey;
   const map = new mapboxgl.Map({
     container: 'map',
-    style: 'mapbox://styles/mapbox/streets-v10',
+    style: 'mapbox://styles/nicode80/ckhwgqtuw0xvk1aoe3yvx7g9k',
```
Comment (mdaudiffret) [inline app/javascript/plugins/init_mapbox.js:11]: ton style est bien publique?

File: app/controllers/pages_controller.rb (hunk 1 above)
Comment (AshFromAvalon) [reply app/controllers/pages_controller.rb]: Agreed with @mdaudiffret , it's campaigns_path

File: app/policies/mission_policy.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+class MissionPolicy < ApplicationPolicy
+  class Scope < Scope
+    def resolve
+      scope.all
+    end
+  end
+
+  def index?
```
Comment (AshFromAvalon) [inline app/policies/mission_policy.rb:8]: @Nicode80 ,I don't think need to enable index this way (it uses authorize). If go to `app/controller/applications_controller.rb` you'll see that auhtorize is not set for index. Index only uses scope policy.

File: config/routes.rb [hunk 4]
```diff
@@ -6,4 +6,5 @@
   resources :materials do
     resources :instructions, only: [:create, :new, :edit, :destroy, :update]
   end
+  resources :missions, only: [:index]
```
Comment (AshFromAvalon) [inline config/routes.rb]: Like @mdaudiffret said, Map will be on Campaigns index, not missions

File: app/controllers/pages_controller.rb (hunk 1 above)
Comment (Nicode80) [reply app/controllers/pages_controller.rb]: My bad. I update that

**Review by mdaudiffret — APPROVED** [review]

**Pull request merged**